plot_far_dipolar_fields.py

```diff
@@ -43,12 +44,14 @@
               'L5Pyr_dend_gnabar_hh2': 0.0}
 l5p = pyramidal(pos=(0, 0, 0), cell_name='L5Pyr', override_params=silence_hh)
 
+_IMEM_HANDLER = _TransmembraneCurrentHandler(pc=_PC, cvode=_CVODE)
```

Comment:
lower-case?
```suggestion
_imem_handler = _TransmembraneCurrentHandler(pc=_PC, cvode=_CVODE)
```

Between, I haven't looked carefully at the `_TransmembraneCurrentHandler` code yet. Could it work if it was a function instead of class? (you saw that coming, didn't you? ;-)

Reply:
I did :) I think it needs to be a class to be persistent and have the callback, but I could be wrong?